### PR TITLE
UPSTREAM: 82189: add conditions for remaining object totals during ns termination (fixes integration test)

### PR DESCRIPTION
**Pull request by p0lyn0mial**
fixes the broken integration test.

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
/hold

I deliberately broke a unit test just to see if it is run in `CI` - https://github.com/openshift/origin/pull/23830/commits/e2221b167a6c5d4c8b2b12c7212d42f041be98ec

**Comment by p0lyn0mial** [comment]
/hold cancel

k8s unit tests are executed - https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/23830/pull-ci-openshift-origin-master-unit/9811

File: vendor/k8s.io/kubernetes/test/integration/namespace/ns_conditions_test.go [hunk 1]
```diff
@@ -98,10 +97,10 @@ func TestNamespaceCondition(t *testing.T) {
 			if condition.Type == corev1.NamespaceDeletionContentFailure && condition.Message == `All content successfully deleted, may be waiting on finalization` {
 				conditionsFound++
 			}
-			if condition.Type == corev1.NamespaceContentRemaining && condition.Message == `Some resources are remaining: deployments.apps has 1 resource instances` {
+			if condition.Type == corev1.NamespaceContentRemaining && condition.Message == `Some resources are remaining: deployments.apps has 1 resource instances, deployments.extensions has 1 resource instances` {
 				conditionsFound++
 			}
-			if condition.Type == corev1.NamespaceFinalizersRemaining && condition.Message == `Some content in the namespace has finalizers remaining: custom.io/finalizer in 1 resource instances` {
+			if condition.Type == corev1.NamespaceFinalizersRemaining && condition.Message == `Some content in the namespace has finalizers remaining: custom.io/finalizer in 2 resource instances` {
```
Comment (p0lyn0mial) [inline vendor/k8s.io/kubernetes/test/integration/namespace/ns_conditions_test.go]: our fork has `extentions` group enabled.

File: vendor/k8s.io/kubernetes/test/integration/namespace/ns_conditions_test.go [hunk 2]
```diff
@@ -98,10 +97,10 @@ func TestNamespaceCondition(t *testing.T) {
 			if condition.Type == corev1.NamespaceDeletionContentFailure && condition.Message == `All content successfully deleted, may be waiting on finalization` {
 				conditionsFound++
 			}
-			if condition.Type == corev1.NamespaceContentRemaining && condition.Message == `Some resources are remaining: deployments.apps has 1 resource instances` {
+			if condition.Type == corev1.NamespaceContentRemaining && condition.Message == `Some resources are remaining: deployments.apps has 1 resource instances, deployments.extensions has 1 resource instances` {
```
Comment (p0lyn0mial) [inline vendor/k8s.io/kubernetes/test/integration/namespace/ns_conditions_test.go]: this is a side effect of having multiple versions of the same resource, which can be misleading.

**Comment by p0lyn0mial** [comment]
/hold

I think that the commit message is wrong. It should be `<carry>`? I'm right?

**Comment by p0lyn0mial** [comment]
/hold

I need to fix the failing `API data in etcd should be stored at the correct location and version for all resources ` test

**Comment by p0lyn0mial** [comment]
this goes into `4.2`, master is `4.3`

**Comment by eparis** [comment]
are we still working on this PR?

**Comment by p0lyn0mial** [comment]
closing this in favour of https://github.com/openshift/origin/pull/23910

**Pull request closed without merging**